src/openrct2-win/openrct2-win.cpp

```diff
@@ -40,6 +40,7 @@ int wmain(int argc, wchar_t** argvW, [[maybe_unused]] wchar_t* envp)
     std::vector<const char*> argv;
     std::transform(
         argvStrings.begin(), argvStrings.end(), std::back_inserter(argv), [](const auto& string) { return string.c_str(); });
+    argv.push_back(nullptr);
```

Comment:
A comment explaining why this is added would be nice. Otherwise, it's not clear why this is here.